lotus-soup/init.go

```diff
@@ -48,5 +48,5 @@ func init() {
 	build.UpgradeLiftoffHeight = -3
 	// We need to _run_ this upgrade because genesis doesn't support v2, so
 	// we run it at height 0.
-	build.UpgradeActorsV2Height = 0
```

Comment:
do we not want this?

Reply:
I don't know, this is based on the payment channel PR, this is why this is here.

@willscott see https://github.com/filecoin-project/oni/pull/264

---

P.S. I opened this PR way earlier as I need one to experiment with Github status checks as well.